File: 프로그래머스/1/12935. 제일 작은 수 제거하기/제일 작은 수 제거하기.cpp
#include <string>
#include <vector>
#include <algorithm>

using namespace std;

vector<int> solution(vector<int> arr) {
    vector<int> r=arr;
    int min=*min_element(r.begin(),r.end());
    r.erase(remove(r.begin(),r.end(),min),r.end());
    if(r.size()==0)
        r.push_back(-1);
    return r;
}

File: 프로그래머스/1/12935. 제일 작은 수 제거하기/제일 작은 수 제거하기.java
import java.util.Arrays;
class Solution {
    public int[] solution(int[] arr) {
        int idx=0, len=arr.length, rlt[] = new int[len-1];
        if(rlt.length<1)
            return new int[]{-1};
        int min=Arrays.stream(arr).min().getAsInt();
        for(int i=0;i<len;i++) {
            if(arr[i]!=min) {
                rlt[idx++]=arr[i];
            }
        }    
        return rlt;
    }
}